pkg/endpoint/endpoint.go

```diff
@@ -225,7 +225,7 @@ type Endpoint struct {
 
 	// k8sPorts contains container ports associated in the pod.
 	// It is used to enforce k8s network policies with port names.
-	k8sPorts policy.NamedPortsMap
+	k8sPorts policy.NamedPortMap
```

Comment:
Is there any significance to policy.NamedPortMap vs policy.NamedPortsMap.
You've renamed it in endpoint.go but not in epinfo.go.

Reply:
`policy.NamedPortsMap` is now an interface that abstracts the implementation (simple map, used for ingress that only needs to consider named ports of the POD itself, or a multimap used for egress that needs to map a port name of destination PODs).